cmake/features.cmake

```diff
@@ -162,7 +162,15 @@ else()
     set(ENABLE_SYSTEM_FLATBUFFERS_DEFAULT ON)
 endif()
 
-ov_dependent_option (ENABLE_SYSTEM_TBB  "Enables use of system TBB" OFF "THREADING MATCHES TBB" OFF)
+# use system TBB only for Debian / RPM packages
+if(CPACK_GENERATOR MATCHES "^(DEB|RPM)$")
```

Comment:
Is this modification due to (https://github.com/openvinotoolkit/openvino/blob/4b0868c98a32d888fe240d86669101ff32e07681/src/cmake/install_tbb.cmake#L98)?
Do other cpack types (like `CONDA-FORGE|BREW|CONAN|VCPKG`) need to be added here?

Reply:
good catch, thank you!